augly/image/functional.py

```diff
@@ -911,6 +924,7 @@ def meme_format(
 
     image = imutils.validate_and_load_image(image)
     func_kwargs = imutils.get_func_kwargs(metadata, locals())
+    src_mode = image.mode
```

Comment:
i think you forgot to use  this `src_mode` on line 969

Reply:
Good catch!